README.md

```diff
@@ -0,0 +1,7 @@
+# FaaS-environment-config
+A Terraform module for deploying Lambda functions with an API gateway from an S3 bucket
+
+There are two branches that should be considered *production* branches:
+
+- `develop` - original FaaS development track this branch.
```

Comment:
default branch, with master being the main production branch

Reply:
Have made readme clearer and more accurate